src/mui/detail/TabbedShowLayout.js

```diff
@@ -67,16 +65,11 @@ TabbedShowLayout.propTypes = {
     resource: PropTypes.string,
     basePath: PropTypes.string,
     translate: PropTypes.func,
+    version: PropTypes.number,
 };
 
 TabbedShowLayout.defaultProps = {
     contentContainerStyle: { borderTop: 'solid 1px #e0e0e0' },
 };
 
-const enhance = compose(
```

Comment:
This probably came from a bad copy/paste of the `TabbedFormLayout`. There is no `initialValues` used in the component.